CppCoreGuidelines.md

```diff
@@ -4035,7 +4064,7 @@ If the `Handle` owns the object referred to by `s` it must have a destructor.
 
 Independently of whether `Handle` owns its `Shape`, we must consider the default copy operations suspect:
 
-    Handle x {*new Circle{p1, 17}};  // the Handle had better own the Circle or we have a leak
+    Handle x {*new Circle{p1, 17}};  // causes a leak if the Handle is not a Circle
```

Comment:
Similar to above, some of the meaning has changed. The old comment discussed ownership. Maybe it's better this way but worth thinking about.